README.md

```diff
@@ -1,68 +1,252 @@
+                    alert(data.Error);
+                } else {
+                    const movieData = data.Search;  
+                    this.setState({movie: movieData});
+                    this.setState({isPerformingSearch: true});
+                };
+            }) 
+        };
+    }
+```
+
+
+### Search Results ###
+If a search has not been run, `<SearchResults/>` renders responsive text echoing the current value of the input field and number of results. 
+If a search has been run, `<SearchResults/>` receives MovieApp.state.movie as props and renders as text and imagery.
+The search results display image, title, release date (year) and ImDb ID no for each movie by mapping through the array of nested objects. A hyperlink links to that movie's ImDb page.
+
+```javascript
+const movies = newArray.map((movie, index) => { 
+            return ( 
+                <div 
+                    key={movie.imdbID} 
+                    data-testid={ "SK" + indexPlus }>   
+                ...
+            )
+}                        
+```
+
+<img src="./images/movieInfoScreenshot.png" width="450" alt="screenshot3">
+
+### Sort Criteria Dropdown ###
+If a search has been run, `<SortCriteriaDropdown/>` renders static text and a dropdown list. Note that when only one search result is returned, the dropdown list is disabled. 
+
+<img src="./images/searchResultsScreenshot.png" width="450" alt="screenshot4">
+
+The static text allows the user to see if the number of results returned is as per their selection. 
+If it is, the text is purple. If it is not, the text will be pink.
+
+<img src="./images/lessResultsScreenshot.png" width="450" alt="screenshot5">
+
+The dropdown list allows movie results to be sorted by several categories such as 'Title A-Z'. 
+
+<img src="./images/sortDropdownScreenshot.png" width="450" alt="screenshot6">
+
+When a user selects an option, this updates MovieApp.state.sort with the relevant Object from SortCriteriaObjects.js.
+ `<SearchResults/>` receives this Object as props, sorts the results accordingly and re-renders.
+
+### Sort Criteria Objects.js ###
+A Module containing enumerated Objects, imported and used by `<MovieApp/>` and `<SortCriteriaDropdown/>`
+The Object holds information for each sort category, including a callback comparator function ready to be passed as an argument to array.sort() to sort search results. 
+
+```javascript
+function sortMovies (a, b, propertyName) {
+    const movieA = a[propertyName]; 
+    const movieB = b[propertyName];         
+    let comparison = 0;
+
+    if (movieA > movieB) {
+        comparison = 1
+    } else if (movieA < movieB) {
+        comparison = -1
+    };
+    return comparison;
+}
+
+const sortCriteria= {
+
+        IMBDID : {
+            name: "IMBDID",
+            userMessage: "imdbID no.", 
+            category: "imdbID",
+            comparator(a,b) {
+                return sortMovies(a, b, "imdbID");
+            }
+        },
+```
+
+
+
+## Getting Started
```

Comment:
This should probably be the second section in the README (nearly at the top of the page)